doc/user_guide/customization.rst

```diff
@@ -290,3 +290,88 @@ You can remove the legend entirely by submitting a null value.
       y='petalLength',
       color=alt.Color('species', legend=None),
   )
+
+Adjusting the width of Bar Marks
+--------------------------------
+The width of the bars in a bar plot are controlled through the `size` property in the `mark_bar()`:
+
+.. altair-plot::
+
+  import altair as alt
+  import pandas as pd
+
+  data = pd.DataFrame({'name': ['a', 'b'], 'value': [4, 10]})
+
+  alt.Chart(data).mark_bar(size=10).encode(
+      x='name:O',
+      y='value:Q'
+  )
+
+But since `mark_bar(size=10)` only controls the width of the bars, it might become possible that the width of the chart is not adjusted accordingly:
+
+.. altair-plot::
+
+  import altair as alt
+  import pandas as pd
+
+  data = pd.DataFrame({'name': ['a', 'b'], 'value': [4, 10]})
+
+  alt.Chart(data).mark_bar(size=30).encode(
+      x='name:O',
+      y='value:Q'
+  )
+
+The width of the chart containing the bar plot can be controlled through two mechanisms:
+
+1. Setting the `width` of chart, so the width of the bars are adjusted to fit the width of the chart.
+
+2. Setting the `rangeStep` of the bars, so the width of the chart becomes the number of bars multiply the `rangeStep`.
```

Comment:
I would clarify explicitly here that `rangeStep` means the width (in pixels) that the chart allocates for each bar.